wdl2cwl/main.py

```diff
@@ -43,57 +133,65 @@ def main(argv: List[str]) -> str:
     ast = WdlV1_1ParserVisitor()  # type: ignore
     ast.walk_tree(tree)  # type: ignore
 
+    input_types: List[str] = []
+    input_names: List[str] = []
+    input_values: List[str] = []
+
+    for i in ast.task_inputs:
+        input_types.append(i[0])
+        input_names.append(i[1])
+        input_values.append("None")
+
+    for i in ast.task_inputs_bound:
+        input_types.append(i[0])
+        input_names.append(i[1])
+        input_values.append(i[2])
```

Comment:
This section & https://github.com/common-workflow-lab/wdl-cwl-translator/pull/17/files#diff-32fbb383f6a2270c651e2ddaeb7cea4d5f1121735dc0d4f1dc8980d8d692158aR68 are also not being called by your tests, so can you either remove it, or amend your test to have this WDL feature